ports/rp2/boards/SIL_RP2040_SHIM/board.json

```diff
@@ -0,0 +1,20 @@
+{
+    "deploy": [
+        "deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Dual-core",
+        "RGB LED",
+        "External Flash",
+        "USB"
+    ],
+    "images": [
+        "../images/rp2040-shim-product.jpg"
```

Comment:
The file name has to match the name in a directory boards/SIL_RP2040_SHIM in the micropython-media repository. You have to push the image to that place,

Reply:
Oh that's where images go, I had been wondering why I couldn't find where any other boards had their images stored.  I pushed a PR to that repo and updated the `board.json` reference.